Makefile

```diff
@@ -1,5 +1,26 @@
+staking-bank-production: 
+	cd ./staking-bank; cargo concordium build -e --out staking_bank.wasm.v1 -- --features production;
+
+staking-bank-development: 
+	cd ./staking-bank; cargo concordium build -e --out staking_bank.wasm.v1 -- --features development;
+
+staking-bank-sandbox: 
+	cd ./staking-bank; cargo concordium build -e --out staking_bank.wasm.v1 -- --features sandbox;
+
+build-contracts:
+	cd registry; cargo concordium build -e --out registry.wasm.v1; cd ../umbrella-feeds; cargo concordium build -e --out umbrella_feeds.wasm.v1; cd ../dummy-contract; cargo concordium build -e --out dummy-contract.wasm.v1;
+
+build-all-production: 
+	make build-contracts; make staking-bank-production;
+
+build-all-development:
+	make build-contracts; make staking-bank-development;
+
+build-all-sandbox:
+	make build-contracts; make staking-bank-sandbox;
```

Comment:
Wouldn't it make more sense to use the dependencies in Makefile rather than writing a bash script?

So for example
```
build-all-sandbox: build-contracts staking-bank-sandbox
```

It's odd to run make from inside a Makefile.